bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx

```diff
@@ -440,6 +427,7 @@ class AudioModal extends Component {
       showPermissionsOvelay,
       isIOSChrome,
       closeModal,
+      isIEOrEdge,
```

Comment:
Do we still need this given that we have https://github.com/bigbluebutton/bigbluebutton/pull/7303 ?

Reply:
IE will never reach it, but Edge will